### ISSUE TITLE
torch.size question in NavPedPreNet()

### ISSUE THREAD

**Issue by SpikeJishuo** [DRL-Navigation/DDRL4NAV#1]
https://github.com/DRL-Navigation/DDRL4NAV/blob/8d5281517f188fd69c1306240ddd83d2f6c3c3dc/USTC_lab/nn/nav_encoder.py#L72

Could you plz tell me what does state[2] stand for in pedstrians exsisting scenario?  
I got torch.size([1, 96, 48]) of `torch.cat([state[0], state[2]], axis=1))) `instead of ([64, 4, 3, 3]) .

**Comment by SnallQiu** [comment]
the index of state here is defined in `https://github.com/DRL-Navigation/img_env/blob/master/envs/wrapper/filter_states.py`

-   state[0]: means sensor_maps, [batch, 1, 48, 48], egocentric sensor map.
-   state[2]: ped_maps, [batch, 3, 48, 48] , egocentric 3-channel pedestrians map, which means x，y velocity and position of the nearby pedestrians.   

  For more details of the state, you can see readme of https://github.com/DRL-Navigation/img_env.

**Comment by SnallQiu** [comment]
> https://github.com/DRL-Navigation/DDRL4NAV/blob/8d5281517f188fd69c1306240ddd83d2f6c3c3dc/USTC_lab/nn/nav_encoder.py#L72
> 
> Could you plz tell me what does state[2] stand for in pedstrians exsisting scenario? I got torch.size([1, 96, 48]) of `torch.cat([state[0], state[2]], axis=1))) `instead of ([64, 4, 3, 3]) .

the index of state here is defined in `https://github.com/DRL-Navigation/img_env/blob/master/envs/wrapper/filter_states.py`

-   state[0]: means sensor_maps, [batch, 1, 48, 48], egocentric sensor map.
-   state[2]: ped_maps, [batch, 3, 48, 48] , egocentric 3-channel pedestrians map, which means x，y velocity and position of the nearby pedestrians.   

  For more details of the state, you can see readme of https://github.com/DRL-Navigation/img_env.
